### PR TITLE
add method waitOnTrafficToFinish

### PR DESCRIPTION
**Pull request by Emil-Gustafsson**
Signed-off-by: Emil Gustafsson <emil.gustafsson@ericsson.com>

### CODE REVIEW

**Comment by Emil-Gustafsson** [comment]
not sure if this should be here or in a higher layer

File: src/main/java/com/cisco/trex/stateless/TRexClient.java [hunk 1]
```diff
@@ -380,6 +381,68 @@ public void removeRxFilters(int portIndex, int profileId) {
...
+    }
+    while (portTrafficMap.containsValue(true)) {
+      for (Entry<Port, Boolean> entry : portTrafficMap.entrySet()) {
+        if (entry.getValue()) { // port is still true, meaning still running traffic
+          // set port to false if traffic is stopped(IDLE)
+          entry.setValue(!getPortStatus(entry.getKey().getIndex()).get().getState().equals("IDLE"));
+        }
+      }
+      if (System.currentTimeMillis() > endTime) {
+        break;
```
Comment (leomaatEric) [inline src/main/java/com/cisco/trex/stateless/TRexClient.java]: 1. How to handle the timeout happening? should it throw an exception or logging it.
2. Does not the "timeout" contain the "rxDelayMs" time? it only mean the timeout setting for sending packets on ports.

Comment (Emil-Gustafsson) [reply src/main/java/com/cisco/trex/stateless/TRexClient.java]: i think we could add a logging of it atleast.
rx delay ms is the time after port have stopped sending traffic and we wait to remove the rx filter, so it means the time it takes for last packet to arrive at destination port.
Another thing that is missing here is a short wait between port status checks.

Comment (leomaatEric) [reply src/main/java/com/cisco/trex/stateless/TRexClient.java]: fair enough, please go with update then merging code.

**Pull request merged**